webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -159,6 +168,15 @@
 		<url-pattern>/*</url-pattern> 
 	</filter-mapping>
 
+	<filter>
+		<filter-name>CSRFGuard</filter-name>
+		<filter-class>org.owasp.csrfguard.CsrfGuardFilter</filter-class>
+	</filter>
+	<filter-mapping>
+		<filter-name>CSRFGuard</filter-name>
+		<url-pattern>/*</url-pattern>
```

Comment:
Maybe it's better to make this something like:

```java
<url-pattern>*.htm</url-pattern>
<url-pattern>*.html</url-pattern>
<url-pattern>*.page</url-pattern>
<url-pattern>*.list</url-pattern>
<url-pattern>*.form</url-pattern>
```

I tend to prefer white-listing over black-listing (like this), but I don't really see a good alternative here... That should capture:

* RefApp 2.x forms
* 1.x (Legacy UI) forms
* OWAs

Reply:
@ibacher did you get a chance to evaluate the pros and cons of doing this from the csrfguard.properties configuration file? That is, assuming we put it in the application data folder just like the runtime properties file.

Reply:
@dkayiwa Admittedly, I did not read all the configuration options in the configuration file. Yep, doing things there is much better! (Proper support for an allow-list rather than a deny-list).